app/components/finance/ecf/breakdown_summary.html.erb

```diff
@@ -74,15 +74,13 @@
 
 <table class="govuk-table payment-breakdown-table">
   <tbody>
-    <!--TODO: Add VAT-->
     <tr class="govuk-table__row">
       <th scope="row" class="govuk-table__cell govuk-!-font-weight-regular">VAT</th>
-      <td class="govuk-table__cell govuk-table__cell--numeric"><%= number_to_pounds 0 %></td>
+      <td class="govuk-table__cell govuk-table__cell--numeric"><%= number_to_pounds total_vat(@breakdown) %></td>
```

Comment:
VAT is going to be optional per provider. We probably shouldn't show it for providers that aren't charging it. I think we probably need to have a flag against the lead provider to determine if they are or aren't VAT charging.

Reply:
Added the column to lean provider